standalone-metastore/metastore-server/src/docker/build.sh

```diff
@@ -101,6 +101,7 @@ fi
 
 cp "$CACHE_DIR/hadoop-$HADOOP_VERSION.tar.gz" "$WORK_DIR/"
 cp -R "$SOURCE_DIR/metastore-server/src/docker/conf" "$WORK_DIR/"
+cp -R "$SOURCE_DIR/metastore-server/src/main/resources/metastore-site.xml" "$WORK_DIR/conf/"
```

Comment:
Shoud we drop the `https://github.com/apache/hive/blob/master/standalone-metastore/metastore-server/src/docker/conf/metastore-log4j2.properties` as well?

We can use this `https://github.com/apache/hive/blob/master/standalone-metastore/metastore-server/src/main/resources/metastore-log4j2.properties` uniformly.

Reply:
cc @deniskuzZ

Reply:
yes, 100%. 
i didn't notice that we already had these templates in HMS resources